static/src/javascripts/projects/commercial/modules/prebid/slot-config.js

```diff
@@ -10,14 +12,17 @@ import config from 'lib/config';
 
 import type { PrebidSlot } from 'commercial/modules/prebid/types';
 
-const filterByAdvertId = (
-    advertId: string,
+const defaultMatchesDfpAdvert = (pbs: PrebidSlot, dfpa: Advert) =>
```

Comment:
Since this isn't used anywhere else and isn't tested, would it be simpler to inline it?